src/formpack/schema/fields.py

```diff
@@ -209,6 +217,8 @@ def from_json_definition(cls, definition, hierarchy=None,
             # legacy type, treat them as text
             'select_one_external': partial(TextField, data_type=data_type),
             'cascading_select': partial(TextField, data_type=data_type),
+            # handle legacy `select one`
+            'select one': FormChoiceField,
```

Comment:
Is this necessary? If it gets to this point, would the field already be changed to `select_one`?

Reply:
This is to handle forms that still have `select one` because of the bug, but all future forms should now become `select_one`. An old form won't export with the choice labels unless that's included (I think it broke the auto-report too without it).

Reply:
I wrote in [another comment](https://github.com/kobotoolbox/formpack/pull/251/files/4bec4814ea001c7d60c9d0c13adc67b582d8c6ce#r657579733) but I think we have managed to avoid having aliases at this stage. This is a slippery slope and if we don't prevent aliases from getting this far, then we have to start accommodating more of them.